### DISCUSSION TITLE
v2024.12.115

### DISCUSSION THREAD

**Discussion by zyrouge** [zyrouge/symphony#710] [Announcements]
## What's Changed

* Ability to modify Album cover quality. (#118)
* Added support for gapless playback. (#594)
* Ability to modify grid sizes. (https://github.com/zyrouge/symphony/pull/661, https://github.com/zyrouge/symphony/pull/673 by @sevonj)
* Added sort by album year. (https://github.com/zyrouge/symphony/pull/674 by @sevonj)
* Added ability to disable case-sensitive sorting. (#445)
* Display album duration in Album view. (https://github.com/zyrouge/symphony/pull/666 by @sevonj)
* Display album year in Album view. (https://github.com/zyrouge/symphony/pull/681 by @sevonj)
* Added song duration setting. (https://github.com/zyrouge/symphony/pull/698 by @sevonj)
* Fixed sort direction icon. (https://github.com/zyrouge/symphony/pull/672 by @sevonj)
* Fixed queue clear causing `OutOfBoundsException`. (https://github.com/zyrouge/symphony/pull/682 by @sevonj)
* Fixed playlist addition/deletion issue. (#689)
* Sorting by track number now considers disc number.
* Fixed an issue where activity configuration changes glitches the interface. (#593)
* Albums now don't consider year to be a factor. (#655)
* Fixed missing buttons in traditional layout. (#598)
* Added Finnish translations. (https://github.com/zyrouge/symphony/pull/668 by @sevonj)
* Updated Polish translation. (https://github.com/zyrouge/symphony/pull/650, https://github.com/zyrouge/symphony/pull/659, https://github.com/zyrouge/symphony/pull/686, https://github.com/zyrouge/symphony/pull/695, https://github.com/zyrouge/symphony/pull/699 by @rehork)

## New Contributors
* @sevonj made their first contribution in https://github.com/zyrouge/symphony/pull/661

**Full Changelog**: https://github.com/zyrouge/symphony/compare/v2024.11.114...v2024.12.115

<hr /><em>This discussion was created from the release <a href='https://github.com/zyrouge/symphony/releases/tag/v2024.12.115'>v2024.12.115</a>.</em>

**Comment by CJottawa** [comment]
Zyrouge
Been a while commenting on Symphony. 
Installed the apk v2024.12.115 and it does not recognize my music directories (ie categories) which each contain music files. 
Symphony worked before and loved it. Even tried the install from the Google Play store. De-installed the version I had and tried both ways again. NO luck.
:(

**Reply by zyrouge** [reply]
Try this. https://github.com/zyrouge/symphony/wiki/User-Guide#adding-media-folders

**Reply by CJottawa** [reply]
Thanx for this Zyrouge - worked just fine
New issue is that the album art in each Music folder is not being recognized. There is only 1 jpg file in each folder. I tried doing your above suggestion for several of the Music Folders as well and album art issue remained.
There has not been a problem before this.
Chris

**Reply by CJottawa** [reply]
Oh By the way - I also googled the heck out of this one and found zero

**Reply by zyrouge** [reply]
Using jpg files as cover is not yet supported.

**Reply by CJottawa** [reply]
I used them for quite a while with Symphony and suddenly not?

**Reply by CJottawa** [reply]
What is the format this is supported?

**Reply by zyrouge** [reply]
Only covers embedded within song tag is supported.
